src/Game/Systems/CapturePointArrowHUDSystem.cpp

```diff
@@ -0,0 +1,137 @@
+#include "Systems/CapturePointArrowHUDSystem.h"
+
+CapturePointArrowHUDSystem::CapturePointArrowHUDSystem(SystemParams params)
+    : System(params)
+    , ImpureSystem()
+{
+    EVENT_SUBSCRIBE_MEMBER(m_ECapturedEvent, &CapturePointArrowHUDSystem::OnCapturePointCaptured);
+}
+
+
+void CapturePointArrowHUDSystem::Update(double dt)
+{
+    bool LoadCheck = true;
```

Comment:
Please have a look at the code style guide. LoadCheck, ArrowHUDs, CapturePoints does not follow it. :)